crates/common/src/lib.rs

```diff
@@ -74,3 +80,94 @@ pub fn to_wormhole_address(address: Address) -> B256 {
     bytes[12..].copy_from_slice(address.as_slice());
     B256::from(bytes)
 }
+
+/// Converts a Wormhole format B256 address to an Ethereum Address.
+pub fn from_wormhole_address_align(wormhole_addr: B256) -> Result<Address, String> {
```

Comment:
Should remove the `pub` and possibly put this inside the tests module as this is the only place it is used